### ISSUE TITLE
o.pickProjectAndStart not found

### ISSUE THREAD

**Issue by Velmrok** [dotnet/vscode-csharp#6362]
## Environment data
`dotnet --info` output:
![image](https://github.com/dotnet/vscode-csharp/assets/120123931/8e012de0-decc-470c-8b70-986718bd161c)

VS Code version: 
C# Extension version: 2.1.2

## Steps to reproduce

ctrl+shift+p omnisharp select project

## Expected  behavior
Dialog to select project

## Actual behavior
Notification: command 'o.pickProjectAndStart' not found

**Comment by dibarbet** [comment]
@Velmrok are you using the `dotnet.server.useOmnisharp` setting?

If not, this command is not expected to exist in the Roslyn LSP version.

**Comment by Velmrok** [comment]
Yes, i do use. This happened randomly when i was making new project
